plugins/pipelines/app/models/concerns/samson_pipelines/stage_concern.rb

```diff
@@ -1,15 +1,26 @@
 module SamsonPipelines::StageConcern
-  # Return true if any stages in the pipeline are marked production
-  def production?
```

Comment:
if you remove production? then `def deploy_buddy` needs to check for `deploy_requires_approval?` ... same for `deploy.production?` and `latest_approved_deploy` and `validate_bypass_used_correctly` (I'll remove that in another PR ... so don't worry about validate_bypass_used_correctly) ... also the `production?` would be  wrong now since the stage does to production ...